File: easy/42/42.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

# started at: Tuesday, 12. November 2013 12:32PM


def main():
    answer = 0
    while answer != 42:
        answer = int(raw_input())
        if answer!=42:
            print answer

if __name__ == '__main__':
    main()

File: easy/42/42.c
 /*
    started at: Tuesday, 12. November 2013 12:32PM
*/

#include <stdio.h>

int main(int argc, char const *argv[])
{   
    int answer=0;

    do
    {   
        scanf("%d",&answer);
        if (answer!=42)
        {
            printf("%d\n",answer); 
        }
    }while(answer!=42);
    
    return 0;
}